### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**

### CODE REVIEW

**Review by Nikyxa — APPROVED** [review]
Good job!

File: cinema/serializers.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("title", instance.title)
+        instance.description = validated_data.get(
+            "description", instance.description
+        )
+        instance.duration = validated_data.çget("duration", instance.duration)
```
Comment (TPolina) [inline cinema/serializers.py:20]: `çget`?

**Review by TPolina — CHANGES_REQUESTED** [review]
Delete `.idea` from Github

**Review by TPolina — APPROVED** [review]
Well done!

**Pull request closed without merging**